cameraserver/src/main/native/include/cameraserver/CameraServer.h

```diff
@@ -17,6 +17,12 @@
 
 #include "cscore.h"
 
+#ifdef USE_RAW_OPENCV
```

Comment:
Should this be prefixed with CS_?  Otherwise this could be namespace pollution.

Reply:
I actually had to remove this, because I found a major flaw in attempting to do it. So just removed it.